redis/default.toml

```diff
@@ -23,6 +23,10 @@ tcp-backlog = 511
 # bind = [] - listen on all interfaces
 bind = []
 
+# By default, run Redis in protected mode.
+# Uncomment to disable.
+# protected-mode = "no"
```

Comment:
commenting out `protected-mode` gives us three possibilities:
- yes
- no
- unset

I'd rather see the default be:
`protected-mode = "yes"`